### PR TITLE
Logging and SSL not dependant on Logstash

### PR DESCRIPTION
**Pull request by serkin**

### CODE REVIEW

File: templates/filebeat.yml.j2 [hunk 1]
```diff
@@ -104,6 +104,7 @@ output:
     # For Packetbeat, the default is set to packetbeat, for Topbeat
     # top topbeat and for Filebeat to filebeat.
     #index: filebeat
+{% endif %}
```
Comment (geerlingguy) [inline templates/filebeat.yml.j2]: The `{% if filebeat_ssl_certificate_file and filebeat_ssl_key_file %}` already applies above, so it wouldn't need to be defined a second time if Logstash isn't present... or am I reading the configuration wrong? The two top level keys are `elasticsearch` and `logstash`, and each has an independent `tls` configuration, and that level in the JSON object hierarchy would mean `tls` gets defined twice for `elasticsearch` if `filebeat_output_logstash_enabled` is false.

Comment (serkin) [inline templates/filebeat.yml.j2]: No. For example you cannot enable **logging** without enabling **logstash**

Comment (serkin) [inline templates/filebeat.yml.j2]: I don't use logstash but i want to enable logging

**Comment by sdarwin** [comment]
Hi @serkin , you were requesting to separate out Logging _and_ SSL.   However it should only be Logging that is moved into another if clause, leaving the SSL sections the way they are.   Perhaps modify this pull request?

File: templates/filebeat.yml.j2 (hunk 1 above)
Comment (ymartin59) [reply templates/filebeat.yml.j2]: OK. So additional "endif" has to be placed after "#curve_types: []  {% endif %}"

**Pull request closed without merging**